atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/queue/WriteInfoPartitioner.java

```diff
@@ -88,10 +89,10 @@ private PartitionInfo getPartitionInfo(WriteInfo write, int shards) {
     private TableMetadataPersistence.SweepStrategy getStrategyFromKvs(TableReference tableRef) {
         try {
             return TableMetadata.BYTES_HYDRATOR.hydrateFromBytes(kvs.getMetadataForTable(tableRef)).getSweepStrategy();
-        } catch (Throwable th) {
-            log.warn("Failed to obtain sweep strategy for table {}. Assuming sweep strategy is CONSERVATIVE.",
-                    LoggingArgs.tableRef(tableRef), th);
-            return TableMetadataPersistence.SweepStrategy.CONSERVATIVE;
+        } catch (Exception e) {
```

Comment:
This seems reasonable. It would be nice to unify behavior to make a general TableMetadata cache along with `ConflictDetectionManager` (and share the already loaded metadata), but that's out of scope for this PR.